dockerenforcer/killer.py

```diff
@@ -96,6 +104,19 @@ def should_be_killed(self, container):
                 logger.error("During execution of rule {0} exception was raised: {1}".format(rule['name'], e))
         return Verdict(False, container, None)
 
+    def all_violated_rules(self, container):
+        if not container:
+            return []
+
+        v_rules = []
+        for rule in self.__rules:
+            try:
+                if rule['rule'](container):
+                    v_rules.append(rule['name'])
```

Comment:
@piontec Am i correct that you suggest changing the `should_be_killed` method? Which should check all violations and save them to the Stat object?

If yes then should we show  all violations in '/' route? Or we can show only first violation as at the moment